losses/boundary_loss.py

```diff
@@ -0,0 +1,103 @@
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+
+
+class BoundaryLoss(nn.Module):
+    """Boundary Loss proposed in:
+    Alexey Bokhovkin et al., Boundary Loss for Remote Sensing Imagery Semantic Segmentation
+    https://arxiv.org/abs/1905.07852
+    """
+
+    def __init__(self, theta0=19, theta=19, ignore_index=None, weight=None):
+        super().__init__()
+
+        self.theta0 = theta0
+        self.theta = theta
+        self.ignore_index = ignore_index
+        self.weight = weight
+
+    def forward(self, pred, gt):
+        """
+        Input:
+            - pred: the output from model (before softmax)
+                    shape (N, C, H, W)
+            - gt: ground truth map
+                    shape (N, H, w)
```

Comment:
Should be capital W.  I don't see any lowercase w in the file so I figured you just meant 'W' ?